recipes/_common.rb

```diff
@@ -21,22 +21,37 @@
 include_recipe 'chef-sugar::default'
 require 'chef/sugar/core_extensions'
 
-# Create the user
-include_recipe 'omnibus::_user'
```

Comment:
@schisamo I can't test anymore, but this was just a way to readdress the creation of the build user and add the login shell in a solaris11 friendly way. It may be removable.. would need to test :(